meerkat/model/metrics.py

```diff
@@ -0,0 +1,184 @@
+import re
```

Comment:
I think we should use `torchmetrics` here. It helps with reproducibility and takes the work out of distributed metrics in the future

Reply:
I have switched to torchmetrics from sklearn for accuracy and f1 scores. IoU and Dice score already use torchmetrics